lokalise/models/v1/audit_log.py

```
@@ -0,0 +1,33 @@
+        "severity_id",
+        "severity",
+        "status_id",
+        "status",
+        "time",
+        "metadata",
+        "actor",
+        "src_endpoint",
+        "http_request",
+        "enrichments",
+        "unmapped",
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)